### DISCUSSION TITLE
Acronyms expansion

### DISCUSSION THREAD

**Discussion by catarinarurbano** [joaomlourenco/novathesis#340] [Q&A] [answered]
First things first, I would like to express my gratitude for all the work you have done, it has been a very good help.

One thing I noticed, when using \gls{} for acronyms, sometimes they appear in the format 'full name (acronym)' (for example, having three acronyms in a row, the first two appear with the description and the last one only as the acronym) and sometimes just the acronym itself. Is it possible to define for all cases that only the acronym appears, without its meaning before?

Thanks in advance.
Have a good weekend.

**Comment by joaomlourenco** [comment, marked as answer]
The package automatically prints "long_version (short_version)" on the first usage, and then only "short_version" on subsequent uses.  But you can force the short version with `\glsxtrshort{…}` and the long version with `glsxtrlong{…}`.   Hope this helps.

**Reply by catarinarurbano** [reply]
Professor, it helps a lot! Thanks again for all your work and for the help you provide us!
